tests/tests/relayer.rs

```diff
@@ -143,6 +141,79 @@ async fn relayer_can_download_logs() {
     eth_node_handle.shutdown.send(()).unwrap();
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn try_quorum_50_percent_syncing() {
+    let mut config = Config::local_node();
+    config.relayer = Some(relayer::Config::default());
+    let relayer_config = config.relayer.as_mut().expect("Expected relayer config");
+    let eth_node = MockMiddleware::default();
+
+    let eth_node = Arc::new(eth_node);
+    let eth_node_handle = spawn_eth_node(eth_node).await;
+
+    relayer_config.relayer = Some(vec![
+        format!("http://{}", eth_node_handle.address)
+            .as_str()
+            .try_into()
+            .unwrap(),
+        "http://localhost:8080".try_into().unwrap(),
```

Comment:
[nit]: Maybe move this to a var with name `unresponsive_eth_node_address` or something like that.